Makefile

```diff
@@ -9,13 +9,15 @@ CI_REPOSITORY ?= https://github.com/src-d/ci.git
 CI_PATH ?= $(shell pwd)/.ci
 CI_VERSION ?= v1
 
+TEST_PRUNE_WITH_IMAGE ?= false
+
 MAKEFILE := $(CI_PATH)/Makefile.main
 $(MAKEFILE):
 	git clone --quiet --branch $(CI_VERSION) --depth 1 $(CI_REPOSITORY) $(CI_PATH);
 
 -include $(MAKEFILE)
 
-GOTEST_INTEGRATION = $(GOTEST) -parallel 1 -count 1 -tags=integration -ldflags "$(LD_FLAGS)"
+GOTEST_INTEGRATION = $(GOTEST) -timeout 20m -parallel 1 -count 1 -tags=integration -ldflags "$(LD_FLAGS)"
```

Comment:
😞

Reply:
What's the reasoning for the change? The new test that removes images should not take long

Reply:
> The new test that removes images should not take long

In fact it wasn't due to this new test. But locally after pruning all the images running the tests took more that 10 minutes and the process was killed 😞. The value of `20m` it's just a "random" value, maybe `15m` is enough, but I don't think it makes much difference.